cli.ts

```diff
@@ -221,46 +245,65 @@ export async function applyMigrations(
   migrate: Migrate,
   migrations: Migration[],
 ): Promise<void> {
-  const unappliedMigrations = migrations.filter((migration) =>
-    !migration.appliedPath
+  const unappliedMigrations = migrations.filter(
+    (migration) => !migration.appliedPath,
   );
-  console.log(
+  logger(
+    "apply",
     `${unappliedMigrations.length || "No"} unapplied migration${
       unappliedMigrations.length !== 1 ? "s" : ""
     }`,
   );
   if (unappliedMigrations.length) {
     for (const migration of unappliedMigrations) {
-      console.log(`Applying migration: ${migration.path}`);
+      logger("apply", `Applying migration: ${migration.path}`);
       await migrate.apply(migration);
     }
-    console.log("Finished applying all migrations");
+    logger("apply", "Finished applying all migrations");
   }
 }
 
 /**
  * Applies all unapplied migrations and outputs the filenames.
  */
 export async function apply(migrate: Migrate): Promise<void> {
-  console.log("Acquiring migrate lock");
+  logger("apply", "Acquiring migrate lock");
   const lock = await migrate.lock();
-  console.log("Acquired migrate lock");
+  logger("apply", "Acquired migrate lock");
 
-  console.log("Checking loaded migrations");
+  logger("apply", "Checking loaded migrations");
   const migrations = await migrate.getUnapplied();
   await applyMigrations(migrate, migrations);
 
-  console.log("Releasing migrate lock");
+  logger("apply", "Releasing migrate lock");
   await lock.release();
-  console.log("Released migrate lock");
+  logger("apply", "Released migrate lock");
 
-  console.log("Done");
+  logger("apply", "Migrations applied successfully");
+}
+
+export function help() {
+  logger(
+    "info",
+    "Migrate allows you to manage your postgres migrations via the CLI and files in your codebase",
+  );
+  logger(
+    "init",
+    "init allows you to initialize your project and creates a migrations table in your database.",
+  );
+  logger(
+    "load",
+    "load allows you to add migrations to your database but not run them",
+  );
+  logger("apply", "apply loads and migrates any unmigrated migrations");
+  logger("list", "list shows you all your migration files and their status");
+  logger("status", "status gives you an overview of your migrations");
```

Comment:
If adding a help command. It should log one message saying "Commands:" Followed by other messages with 2 leading spaces that have the list of commands with the descriptions. Similar to what deno does. You can use padEnd to make all the commands have spacing after them that makes them all line up. No need to do coloring for this.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padEnd
![image](https://github.com/udibo/migrate/assets/3241653/9c002d6e-d4a4-41fe-bbe1-1b8d7eda94ce)